Packages/com.unity.inputsystem/Documentation~/Migration.md

```diff
@@ -34,202 +44,108 @@ There are scripting symbols defined which allow you to use conditional compilati
+[`Input.imeCompositionMode`](https://docs.unity3d.com/ScriptReference/Input-imeCompositionMode.html)|No corresponding API yet.
+[`Input.imeIsSelected`](https://docs.unity3d.com/ScriptReference/Input-imeIsSelected.html)|Get: `Keyboard.current.imeSelected`<br/>Set: `Keyboard.current.SetIMEEnabled(true);`
+[`Input.inputString`](https://docs.unity3d.com/ScriptReference/Input-inputString.html)|Subscribe to the [`Keyboard.onTextInput`](../api/UnityEngine.InputSystem.Keyboard.html#UnityEngine_InputSystem_Keyboard_onTextInput) event:<br/>`Keyboard.current.onTextInput += character => /* ... */;`
 
-```
-Mouse.current.leftButton.isPressed
-Mouse.current.rightButton.isPressed
-Mouse.current.middleButton.isPressed
-
-// You can also go through all buttons on the mouse (does not allocate).
-var controls = Mouse.current.allControls;
-for (var i = 0; i < controls.Count; ++i)
-{
-    var button = controls[i] as ButtonControl;
-    if (button != null && button.isPressed)
-        /* ... */;
-}
-
-// Or look up controls by name.
-((ButtonControl)Mouse.current["leftButton"]).isPressed
-```
+### Mouse
 |Input Manager (Old)|Input System (New)|
 |--|--|
-[`Input.GetMouseButtonDown`](https://docs.unity3d.com/ScriptReference/Input.GetMouseButtonDown.html)|Use [`ButtonControl.wasPressedThisFrame`](../api/UnityEngine.InputSystem.Controls.ButtonControl.html#UnityEngine_InputSystem_Controls_ButtonControl_wasPressedThisFrame) on the corresponding mouse button:
+[`Input.GetMouseButton`](https://docs.unity3d.com/ScriptReference/Input.GetMouseButton.html)<br/>Example: `Input.GetMouseButton(0)`|Use [`isPressed`](../api/UnityEngine.InputSystem.Controls.ButtonControl.html#UnityEngine_InputSystem_Controls_ButtonControl_isPressed) on the corresponding mouse button.<br/>Example: `InputSystem.Mouse.current.leftButton.isPressed`
+[`Input.GetMouseButtonDown`](https://docs.unity3d.com/ScriptReference/Input.GetMouseButtonDown.html)<br/>Example: `Input.GetMouseButtonDown(0)`|Use [`wasPressedThisFrame`](../api/UnityEngine.InputSystem.Controls.ButtonControl.html#UnityEngine_InputSystem_Controls_ButtonControl_wasPressedThisFrame) on the corresponding mouse button.<br/>Example: `InputSystem.Mouse.current.leftButton.wasPressedThisFrame`
+[`Input.GetMouseButtonUp`](https://docs.unity3d.com/ScriptReference/Input.GetMouseButtonUp.html)<br/>Example: `Input.GetMouseButtonUp(0)`|Use [`wasReleasedThisFrame`](../api/UnityEngine.InputSystem.Controls.ButtonControl.html#UnityEngine_InputSystem_Controls_ButtonControl_wasReleasedThisFrame) on the corresponding mouse button.<br/>Example: `InputSystem.Mouse.current.leftButton.wasReleasedThisFrame`
+[`Input.mousePosition`](https://docs.unity3d.com/ScriptReference/Input-mousePosition.html)|Use [`Mouse.current.position.ReadValue()`](../api/UnityEngine.InputSystem.Mouse.html)<br/>__Note__: Mouse simulation from touch isn't implemented yet.
```

Comment:
For a moment I thought the mouse simulation part might be out of date

But I've only sound the revised - "simulating touch from mouse" (rather than mouse from touch)
https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/manual/Touch.html#touch-simulation

^ so this comment is just a pointless note ;)

Reply:
I'd suggest putting a code section with this bit
position = Mouse.current.position.ReadValue();

And keeping the Mouse documentation page link separate.
As I think its more clear what to cut and paste into code.